modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -73,47 +73,46 @@ The following diagram provides a simplified view of the communication between th
 image:basic-rosetta-api-integration.svg[]
 
 As this diagram suggests, the Rosetta node maintains a local copy of the {IC} ledger canister. 
-Periodically, the `+dfinity/rosetta-api+` software running on the Rosetta node updates its local view of the ledger by querying the ledger canister for the tip of the ledger chain, then querying for any missing ledger blocks.
+Periodically, the `+dfinity/rosetta-api+` software running on the Rosetta node updates its local view of the ledger by querying the ledger canister for the latest link of the ledger chain, then querying for any missing ledger blocks.
 The Rosetta node uses the root key of the Internet Computer to ensure that the local copy of the ledger is genuine.
 The integration software also allows you to use the Rosetta node to submit transactions to the {IC} ledger. 
 
-=== Basic integration workflow
+=== The Integration workflow
 
-The following steps summarize the basic operational workflow for transferring ICP tokens if using a Rosetta node to communicate with the {IC} ledger canister.  The description concerns an {IC} principal who authenticates towards the {IC} using a signing key stored in a wallet.
+The following summarizes the basic operational workflow for transferring ICP tokens if you're using a Rosetta node to communicate with the {IC} ledger canister. You must be an {IC} principal who authenticates to the {IC} using a signing key stored in a wallet.
 
-After a user submits a request to the Rosetta node to make a transaction, the request is passed to the integration software running on that node to interact with the {IC} and complete the following steps:
+After a user submits a request to the Rosetta node to make a transaction, the request is passed to the integration software running on that node to interact with the {IC} and completes the following ordered operations:
```

Comment:
"ordered operations" sounds weird to me. maybe "the following operations in order:" ?

Reply:
it's a very common phrase in math books, but I'm fine with the change

Reply:
actually, the order can be implied by the fact that the steps are numbered :)